### PR TITLE
SPOI-9899-kafka-to-database-app-template

### PR DESCRIPTION
**Pull request by yogidevendra**
1. Java code, configurations
2. Application tests

### CODE REVIEW

**Comment by yogidevendra** [comment]
@deepak-narkhede Could you please review this?

File: kafka-to-database-sync/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+### Description
+The Database to Database Sync Application records from configured source PostgreSQL table and writes them in destination PostgreSQL table. It first does a bulk upload from the configured table parallely and continuously polls at configured poll interval for new records and writes them in destination table.
```
Comment (deepak-narkhede) [inline kafka-to-database-sync/README.md]: Please change source to kakfa.

File: kafka-to-database-sync/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+- The application scales linearly with the number of poller partitions.
+- The application is fault tolerant and can withstand node and cluster outages without data loss.
+- The application is also highly performant and can process as fast as the network allows.
+- It is extremely easy to add custom logic to get your business value without worrying about database connectivity and operational details of database poller and database writer.
+- The only configuration user needs to provide is source database connection details, table.
+- This enterprise grade application template will dramatically reduce your time to market and cost of operations.
+
+Import the application from DataTorrent AppHub and launch it to ingest your data from your PostgreSQL table and insert into output PostgreSQL table. Follow the tutorial videos or walkthrough document below to launch the template and add custom logic to process the data during ingestion.
```
Comment (deepak-narkhede) [inline kafka-to-database-sync/README.md]: Description changes to kafka template.

File: kafka-to-database-sync/src/main/resources/META-INF/properties-test.xml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        <name>dt.operator.JdbcOutput.prop.tablename</name>
+        <value>test_event_output_table</value>
+    </property>
+
+    <property>
+        <name>dt.operator.JdbcOutput.port.input.attr.TUPLE_CLASS</name>
+        <value>com.datatorrent.apps.PojoEvent</value>
+    </property>
+
+    <property>
```
Comment (deepak-narkhede) [inline kafka-to-database-sync/src/main/resources/META-INF/properties-test.xml]: Should we keep transform operator properties commented ?

Comment (yogidevendra) [inline kafka-to-database-sync/src/main/resources/META-INF/properties-test.xml]: No. In fact, un-comment them if you have them commented in the other app-templates.

**Pull request merged**